docs/source/advanced_usage.rst

```diff
@@ -46,8 +46,8 @@ auth-related HTTP requests.
 
 .. note::
     QVMs process incoming requests in parallel, while QPUs process them sequentially. If you encounter timeouts while
-    trying to run large numbers of programs against a QPU, try increasing the ``timeout`` parameter on calls to
-    :py:func:`~pyquil.get_qc()` (specified in seconds).
+    trying to run large numbers of programs against a QPU, try increasing the ``execution_timeout`` parameter on calls
+    to :py:func:`~pyquil.get_qc()` (specified in seconds, default is 10 seconds).
```

Comment:
I would omit the default here since that could easily fall stale

Reply:
Good call